### PR TITLE
add small functionality with modal

### PR DESCRIPTION
**Pull request by iamr0man**

### CODE REVIEW

File: step-project-oop/index.html [hunk 1]
```diff
@@ -12,24 +12,56 @@
...
+                        <select name="doctors" id="list-of-doctors">
+                            <option selected disabled>Choose doctor</option>
+                            <option value="cardiologist">Cardiologist</option>
+                            <option value="dentist">Dentist</option>
+                            <option value="therapist">Therapist</option>
+                        </select>
+                    </form>
+                    <ul id="form-inputs">
+                        <li>
+                            <input type="text" id="target" placeholder="Target">
```
Comment (Vlad-Udovenko) [inline step-project-oop/index.html]: Lets do with Js maybe ? (Not in html)

File: step-project-oop/index.html [hunk 2]
```diff
@@ -12,24 +12,56 @@
...
+                    </ul>
                 </div>
                 <div class="modal-footer">
                     <div>Create</div>
                 </div>
             </div>
         </div>
     </div>
-    <script src='dist/js/index.js'></script>
+    <script src='src/js/index.js'></script>
```
Comment (Vlad-Udovenko) [inline step-project-oop/index.html]: here muat be dist, check gulp

Comment (iamr0man) [reply step-project-oop/index.html]: Im check its with debugger, becaise in dist version name function, and variables are ugly

File: step-project-oop/src/scss/style.scss [hunk 3]
```diff
@@ -1,5 +1,13 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```
Comment (Vlad-Udovenko) [inline step-project-oop/src/scss/style.scss]: whаt the fonT ?

Comment (iamr0man) [reply step-project-oop/src/scss/style.scss]: with out sans-serif

**Pull request merged**